config/confighttp/confighttp.go

```diff
@@ -272,7 +272,7 @@ type ServerConfig struct {
 	CORS *CORSConfig `mapstructure:"cors"`
 
 	// Auth for this receiver
-	Auth *configauth.Authentication `mapstructure:"auth"`
+	Auth *AuthConfig `mapstructure:"auth"`
```

Comment:
This is a breaking change for the API but not the end user config right? We should add an API changelog.

Reply:
New changelog added for the API.